src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -491,7 +491,7 @@ private static string TryFindFile(string xmlFilePath, string uriString = null)
             internal static OpenFileCommand DeserializeCore(XmlElement element)
             {
                 XmlElementHelper helper = new XmlElementHelper(element);
-                string xmlFilePath = TryFindFile(helper.ReadString("XmlFilePath"), element.OwnerDocument.BaseURI);
+                string xmlFilePath = TryFindFile(helper.ReadString("FilePath"), element.OwnerDocument.BaseURI);
```

Comment:
Looking a little closer and doing some testing with older files I do not actually hit any breakpoints in this code, and it appears that the older XML files are still loading as expected, but there needs to be some more testing to address the concerns which I have as well.